packages/admin/dashboard/src/i18n/translations/fr.json

```
@@ -0,0 +1,2748 @@
+		"unsavedChangesDescription": "Vous avez des modifications non enregistrées qui seront perdues si vous quittez ce formulaire.",
+		"includesTaxTooltip": "Les prix de cette colonne sont taxés inclus.",
+		"excludesTaxTooltip": "Les prix de cette colonne sont taxés exclusifs.",
+		"noMoreData": "Aucune donnée supplémentaire"
+	},
+	"json": {
+		"header": "JSON",
+		"numberOfKeys_one": "{{count}} clé",
+		"numberOfKeys_other": "{{count}} clés",
+		"drawer": {
+			"header_one": "JSON <0>· {{count}} clé</0>",
+			"header_other": "JSON <0>· {{count}} clés</0>",
+			"description": "Voir les données JSON pour cet objet."
+		}
+	},
+	"metadata": {
+		"header": "Metadata",
+		"numberOfKeys_one": "{{count}} clé",
+		"numberOfKeys_other": "{{count}} clés",
+		"edit": {
+			"header": "Modifier les métadonnées",
+			"description": "Modifier les métadonnées pour cet objet.",
+			"successToast": "Les métadonnées ont été mises à jour avec succès.",
+			"actions": {
+				"insertRowAbove": "Insérer une ligne au-dessus",
+				"insertRowBelow": "Insérer une ligne en dessous",
+				"deleteRow": "Supprimer la ligne"
+			},
+			"labels": {
+				"key": "Clé",
+				"value": "Valeur"
+			},
+			"complexRow": {
+				"label": "Certaines lignes sont désactivées",
+				"description": "Cet objet contient des métadonnées non primitives, telles que des tableaux ou des objets, qui ne peuvent pas être modifiées ici. Pour modifier les lignes désactivées, utilisez l'API directement.",
+				"tooltip": "Cette ligne est désactivée car elle contient des données non primitives."
+			}
+		}
+	},
+	"validation": {
+		"mustBeInt": "La valeur doit être un nombre entier.",
+		"mustBePositive": "La valeur doit être un nombre positif."
+	},
+	"actions": {
+		"save": "Enregistrer",
+		"saveAsDraft": "Enregistrer comme brouillon",
+		"copy": "Copier",
+		"copied": "Copié",
+		"duplicate": "Dupliquer",
+		"publish": "Publier",
+		"create": "Créer",
+		"delete": "Supprimer",
+		"remove": "Retirer",
+		"revoke": "Révoquer",
+		"cancel": "Annuler",
+		"forceConfirm": "Forcer la confirmation",
+		"continueEdit": "Continuer l'édition",
+		"enable": "Activer",
+		"disable": "Désactiver",
+		"undo": "Annuler",
+		"complete": "Terminer",
+		"viewDetails": "Voir les détails",
+		"back": "Retour",
+		"close": "Fermer",
+		"showMore": "Voir plus",
+		"continue": "Continuer",
+		"continueWithEmail": "Continuer avec l'email",
+		"idCopiedToClipboard": "ID copié dans le presse-papier",
+		"addReason": "Ajouter une raison",
+		"addNote": "Ajouter une note",
+		"reset": "Réinitialiser",
+		"confirm": "Confirmer",
+		"edit": "Modifier",
+		"addItems": "Ajouter des éléments",
+		"download": "Télécharger",
+		"clear": "Effacer",
+		"clearAll": "Tout effacer",
+		"apply": "Appliquer",
+		"add": "Ajouter",
+		"select": "Sélectionner",
+		"browse": "Parcourir",
+		"logout": "Déconnexion",
+		"hide": "Masquer",
+		"export": "Exporter",
+		"import": "Importer"
+	},
+	"operators": {
+		"in": "In"
```

```suggestion
		"in": "Dans"
```

I am not sure if we should keep it in English or just use `Dans` as in french

---

It seems an operator so maybe (depending where it is displayed) It should stay in english